clients/python/pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[project]
+name = "chromadb-client"
+dynamic = ["version"]
+
+authors = [
+  { name="Jeff Huber", email="jeff@trychroma.com" },
+  { name="Anton Troynikov", email="anton@trychroma.com" }
+]
+description = "Chroma Client."
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+  'pandas >= 1.3',
+  'requests >= 2.28',
+  'pydantic >= 1.9',
+  'numpy >= 1.21.6',
+  'posthog >= 2.4.0',
+  'typing_extensions >= 4.5.0',
+  'overrides >= 7.3.1',
```

Comment:
One interesting note when thinking about this in terms of the new architecture is that Sqlite is included with Python. So if we wrote our own brute-force vector search impl using numpy/pandas, and used an embedding function accessible via rest (e.g, OpenAI or HF) we could actually run a full embedded/library version of Chroma in the thin client using just this set of deps.